src/modules/esl-carousel/core/esl-carousel.ts

```diff
@@ -199,6 +205,14 @@ export class ESLCarousel extends ESLBaseElement {
     this.renderer && this.renderer.redraw();
   }
 
+  @listen('esl:show:request')
+  protected onHandleFootnotes(e: CustomEvent): void {
```

Comment:
There is a confusion with the name
Probably should be

```suggestion
  protected onShowRequest(e: CustomEvent): void {
```